site/_docs/plugins.md

```diff
@@ -32,10 +32,10 @@ Jekyll generates your site.
 values of the gem names of the plugins you'd like to use. An example:
 
 
-        gems: [jekyll-test-plugin, jekyll-jsonify, jekyll-assets]
+        gems: [jekyll-sass-converter, jekyll-coffeescript, jekyll-watch, jekyll-assets]
```

Comment:
`jekyll-sass-converter` comes pre-packaged with `jekyll` so let's remove that one.